### PR TITLE
fixSlowVideoCharging

### PR DESCRIPTION
**Pull request by Fostrankill**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -15,8 +15,10 @@
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "react-icons": "^4.7.1",
+    "react-lite-youtube-embed": "^2.3.52",
     "react-router-dom": "^6.4.4",
     "react-youtube": "^10.1.0",
```
Comment (cartige) [inline frontend/package.json:20]: remove useless dependency

File: frontend/package.json [hunk 2]
```diff
@@ -15,8 +15,10 @@
...
+    "react-youtube-embed": "^1.0.3",
```
Comment (cartige) [inline frontend/package.json]: is this dependency useful ?

File: package.json [hunk 3]
```diff
@@ -18,5 +18,9 @@
   "devDependencies": {
     "concurrently": "^7.0.0",
     "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "react-lite-youtube-embed": "^2.3.52",
+    "react-youtube-embed": "^1.0.3"
```
Comment (cartige) [inline package.json]: remove dependencies from global package

**Review by cartige — CHANGES_REQUESTED** [review]
Dependencies to remove

File: frontend/package.json (hunk 2 above)
Comment (Fostrankill) [reply frontend/package.json]: I delete it

File: frontend/package.json (hunk 1 above)
Comment (Fostrankill) [reply frontend/package.json:20]: This is the good dependecy

File: package.json (hunk 3 above)
Comment (Fostrankill) [reply package.json]: I delete. thanks ;)

**Review by cartige — APPROVED** [review]

File: frontend/package.json (hunk 1 above)
Comment (LoicBrassart) [reply frontend/package.json:20]: If you use `react-lite-youtube-embed`, you don't need `react-youtube` anymore, right ? Gerald is right, you could remove the old one

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Remove the package in package.json, and it's all good!

**Review by LoicBrassart — APPROVED** [review]
Saw with Romain, ok!

**Pull request merged**